test/testdata/cfg/rescue.rb.desugar-tree-raw.exp

```diff
@@ -20,6 +20,7 @@ ClassDef{
           }
           fun = <U a>
           block = nullptr
+          pos_args = 0
```

Comment:
I added `pos_args` to the raw output so that it's easier to debug cases where it's gotten out of sync with the args.